doc/getting-started/building-from-source.rst

```diff
@@ -54,125 +54,19 @@ source, by following these steps.
 Install build tools and dependencies
 ************************************
 
-ACRN development is supported on popular Linux distributions,
-each with their own way to install development tools:
+ACRN development is supported on Clear Linux distributions,
 
   .. note::
      ACRN uses ``menuconfig``, a python3 text-based user interface (TUI) for
      configuring hypervisor options and using python's ``kconfiglib`` library.
 
-* On a Clear Linux OS development system, install the necessary tools:
+On the Clear Linux OS development system, install the necessary tools:
```

Comment:
I did not remove the operating systems at this time due to potential conflicts in other ACRN documents.